### ISSUE TITLE
Incorrect parsing of input when only 1 question id is given

### ISSUE THREAD

**Issue by mrahul17** [ArtOfCode-/TagSuggestions#1]
The code runs perfectly when more than 1  ids are supplied.

```
rahul@rahul-Aspire-5738:~/Rahul/code/TagSuggestions$ python3 main.py -i [1036 204]
Retrieving page #1 of tags...
Retrieving page #2 of tags...
Finished processing all tags.

Fetching questions #1036, #204...

Tag suggestions for question #1036
video
Press <Enter> to continue.

Tag suggestions for question #204
No tag suggestions for this question.
```

But raises error for a single id

```
rahul@rahul-Aspire-5738:~/Rahul/code/TagSuggestions$ python3 main.py -i [1036]
Retrieving page #1 of tags...
Retrieving page #2 of tags...
Finished processing all tags.

Traceback (most recent call last):
  File "main.py", line 294, in <module>
    main()
  File "main.py", line 51, in main
    if "]" not in sys.argv[index]:
IndexError: list index out of range
```

**Comment by ArtOfCode-** [comment]
Good catch, and I see you've got a fix for it in a [pull request](https://github.com/ArtOfCode-/TagSuggestions/pull/2) too.

**Comment by ArtOfCode-** [comment]
Fixed and closed: pull #2.
